modules/xz/metadata.json

```diff
@@ -16,7 +16,8 @@
         "github:tukaani-project/xz"
     ],
     "versions": [
-        "5.4.5"
+        "5.4.5",
+        "5.4.5.bcr.1"
     ],
     "yanked_versions": {}
```

Comment:
should we yank the previous one?

Reply:
Let's do that as a follow-up so that we can recommend the fixed version.